doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -338,37 +338,37 @@ local html = import 'html.libsonnet';
         },
         {
           name: 'split',
-          params: ['str', 'c'],
+          params: ['str', 'pat'],
           description: |||
-            Split the string <code>str</code> into an array of strings, divided by the single character
-            <code>c</code>.
+            Split the string <code>str</code> into an array of strings, divided by the string
+            <code>pat</code>.
```

Comment:
We should note that up to and including 0.18.0 require the separator to be single character.